ocaml/xapi/xapi_sdn_controller.ml

```diff
@@ -69,3 +69,27 @@ let forget ~__context ~self =
     List.iter
       (fun host -> Helpers.update_vswitch_controller ~__context ~host)
       (Db.Host.get_all ~__context)
+
+let open_port ~__context ~self ~protocol ~port =
+  Pool_features.assert_enabled ~__context ~f:Features.Sdn_port_management ;
```

Comment:
Putting this `Pool_features.assert_enabled` here means that we are checking the feature flag `n` times, where `n` is the number of hosts in the pool. For Citrix Hypervisor, I believe this is unnecessary and that you can move this assert to `message_forwarding.ml` so that we only check it once (however I'm not sure how you check the xapi feature flags at xcp-ng)

Reply:
Indeed! Silly me.